From 21ce1d98a3d7130853bfe1c384b831102c3deaae Mon Sep 17 00:00:00 2001
From: Qingqing Zhou <quic_qqzhou@quicinc.com>
Date: Thu, 5 Sep 2024 13:46:08 +0530
Subject: [PATCH 28/63] firmware: qcom: tzmem: blacklist QCS615 for SHM Bridge

The SHM bridge is not supported on the QCS615 platform. The scm call
of enabling SHM bridge returns error on this platform, this causes
scm driver fails to probe. Blacklist the QCS615 platform to avoid
enabling SHM bridge.

Signed-off-by: Qingqing Zhou <quic_qqzhou@quicinc.com>
---
 drivers/firmware/qcom/qcom_tzmem.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/firmware/qcom/qcom_tzmem.c b/drivers/firmware/qcom/qcom_tzmem.c
index 92b365178235..291310454739 100644
--- a/drivers/firmware/qcom/qcom_tzmem.c
+++ b/drivers/firmware/qcom/qcom_tzmem.c
@@ -76,6 +76,7 @@ static bool qcom_tzmem_using_shm_bridge;
 
 /* List of machines that are known to not support SHM bridge correctly. */
 static const char *const qcom_tzmem_blacklist[] = {
+	"qcom,qcs615",
 	"qcom,sc8180x",
 	"qcom,sdm670", /* failure in GPU firmware loading */
 	"qcom,sdm845", /* reset in rmtfs memory assignment */
-- 
2.34.1

